### PR TITLE
Image added in the UI

### PR DESCRIPTION
**Pull request by harshitagirase**

### CODE REVIEW

**Review by lawzeem — COMMENTED** [review]
image

It seems that we need a default.

File: dam/urls.py [hunk 1]
```diff
@@ -15,6 +15,7 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from . import views, settings
```
Comment (lawzeem) [inline dam/urls.py]: This line causes errors. views cannot be resolved.

**Comment by lawzeem** [comment]
image

Everything looks good so far. Just set in some values for the height and width of the image so that they fit the screen in both desktop and mobile versions.

**Comment by lawzeem** [comment]
Please add back the DAM1.png in img in static. It breaks our homepage.
image

**Comment by harshitagirase** [comment]
Fixed!

**Comment by lawzeem** [comment]
Also the profile in Dashboard is broken.

**Review by lawzeem — DISMISSED** [review]
👍

**Review by anandimous — APPROVED** [review]

**Pull request merged**